test/WebJobs.Script.Tests.Integration/ApplicationInsights/ApplicationInsightsEndToEndTestsBase.cs

```diff
@@ -287,6 +287,7 @@ public async Task Validate_HostLogs()
             ValidateTrace(traces[idx++], "Host lock lease acquired by instance ID", ScriptConstants.LogCategoryHostGeneral);
             ValidateTrace(traces[idx++], "Host started (", LogCategories.Startup);
             ValidateTrace(traces[idx++], "Initializing Host", LogCategories.Startup);
+            ValidateTrace(traces[idx++], "Initializing Warmup Extension", LogCategories.CreateTriggerCategory("Warmup"));
```

Comment:
@fabiocav / @mathewc, this is being logged during host initialization (had to add here to make sure tests pass). I am not sure why it happens only for warmup, and not for timers / files which are also initialized during startup. I did not add any special logging in that extension either.

I will not block this PR as it's only a log entry. But, just want to make sure this is expected and ok?